flink-connectors/flink-connector-files/src/main/java/org/apache/flink/connector/file/table/FileSystemTableSink.java

```diff
@@ -118,19 +118,19 @@
 
     FileSystemTableSink(
             DynamicTableFactory.Context context,
+            ReadableConfig config,
```

Comment:
same comment as above. let's not pass the context but only what is needed by the sink.